### PR TITLE
Turn on partial_success by default.

### PR DESCRIPTION
**Pull request by qingling128**
This depends on https://github.com/GoogleCloudPlatform/google-fluentd/pull/65.

### CODE REVIEW

File: windows-installer/fluent-template.conf [hunk 1]
```diff
@@ -34,6 +32,11 @@
   disable_retry_limit
   # Use multiple threads for processing.
   num_threads 8
+  # Use gRPC by default to avoid issues with AWS and SSL certs.
```
Comment (igorpeshansky) [inline windows-installer/fluent-template.conf]: Why not make it `Use gRPC transport.`, to match the comments in the other configs?

**Comment by igorpeshansky** [comment]
@jacquesliao FYI.

File: windows-installer/fluent-template.conf (hunk 1 above)
Comment (qingling128) [reply windows-installer/fluent-template.conf]: Changed.

**Comment by qingling128** [comment]
No. I'll merge that PR first, rebase this one and merge this one to master. For now, setting the base branch to `lingshi-windows` helps keep the content of changed files clean (otherwise it includes stuff from the other PR). This way I can test from this branch with the gem upgrade included.

**Comment by igorpeshansky** [comment]
Ok, I think I got this now. Let's continue with this one.

**Comment by qingling128** [comment]
Woot woot! Thanks @igorpeshansky!

**Pull request merged**